<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitsAndSetupDialog</class>
 <widget class="QDialog" name="UnitsAndSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Units</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QPlainTextEdit" name="descriptionPlainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="maximumBlockCount">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="systemGroupBox">
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QHBoxLayout" name="systemHorizontalLayout">
      <item>
       <widget class="QRadioButton" name="metricRadioButton">
        <property name="text">
         <string>Metric</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="imperialRadioButton">
        <property name="text">
         <string>Imperial</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="coordinatesLabel">
       <property name="toolTip">
        <string>Coordinates</string>
       </property>
       <property name="text">
        <string>(x, y)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="coordinatesComboBox">
       <property name="toolTip">
        <string>Coordinates</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="areaLabel">
       <property name="toolTip">
        <string>Area</string>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="areaComboBox">
       <property name="toolTip">
        <string>Area</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="modulusLabel">
       <property name="toolTip">
        <string>Modulus</string>
       </property>
       <property name="text">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="modulusComboBox">
       <property name="toolTip">
        <string>Modulus</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="unitWeightLabel">
       <property name="toolTip">
        <string>Unit weight</string>
       </property>
       <property name="text">
        <string>w</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="unitWeightComboBox">
       <property name="toolTip">
        <string>Unit weight</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="loadLabel">
       <property name="toolTip">
        <string>Load</string>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="loadComboBox">
       <property name="toolTip">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="settlementLabel">
       <property name="toolTip">
        <string>Settlement</string>
       </property>
       <property name="text">
        <string>Settlement</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="settlementComboBox">
       <property name="toolTip">
        <string>Settlement</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="temperatureChangeLabel">
       <property name="toolTip">
        <string>Temperature change</string>
       </property>
       <property name="text">
        <string>Temperature Change</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="temperatureChangeComboBox">
       <property name="toolTip">
        <string>Temperature change</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="thermalCoefficientLabel">
       <property name="toolTip">
        <string>Thermal coefficient</string>
       </property>
       <property name="text">
        <string>Thermal Coefficient</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="thermalCoefficientComboBox">
       <property name="toolTip">
        <string>Thermal coefficient</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lengthErrorLabel">
       <property name="toolTip">
        <string>Length error</string>
       </property>
       <property name="text">
        <string>Length Error</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="lengthErrorComboBox">
       <property name="toolTip">
        <string>Length error</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UnitsAndSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UnitsAndSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
